src/StateService/Network/StateRoot.cs

```diff
@@ -10,9 +10,9 @@
 using System;
 using System.IO;
 
-namespace Neo.Plugins.StateService.Storage
+namespace Neo.Plugins.StateService.Network
 {
-    public class StateRoot : IVerifiable, ISerializable
+    class StateRoot : StateMessage, IVerifiable
```

Comment:
Why not send this message directly? I think we don't need to wrap it in an `ExtensiblePayload`, right?

Reply:
Node without will relay `ExtensiblePayload` but doesn't relay StateRoot.

Reply:
Then it shouldn't implement `IVerifiable`. We should verify the `ExtensiblePayload`.

Reply:
StateRoot has the multi-sig of verifiers ExtensiblePayload doesn't.

Reply:
I don't think so. All witness support multiple signatures.

Reply:
Node with stateservice only need to store `version+index+roothash+witness` so verifiers only sign `version+index+roothash` when exchange signatures.
If we put muti-sig in `ExtensiblePayload`, node need to store this payload. We don't need to `Category `, `ValidBlockStart`, `ValidBlockEnd`.
Say if neo p2p protocol doesn't support `Block` message, will you put block in `Data` field of `ExtensiblePayload` or  use `ExtansiblePayload` as block?